python_modules/libraries/dagster-airbyte/dagster_airbyte/asset_defs.py

```diff
@@ -82,6 +89,16 @@ def _build_airbyte_asset_defn_metadata(
         group_name=group_name,
         key_prefix=asset_key_prefix,
         can_subset=False,
+        metadata_by_output_name={
+            table: {
+                "schema": serialize_dagster_namedtuple(
```

Comment:
is this manual serialization step necessary? I would have assumed that this would get automatically serialized by the dagster framework (but could be wrong)

Reply:
Yeah, kicking off the run ends up running into:

```
TypeError: unhashable type: 'list'
  File "/Users/ben/Documents/repos/dagster/python_modules/dagster/dagster/_grpc/impl.py", line 97, in core_execute_run
    recon_pipeline.get_definition()
  File "/Users/ben/Documents/repos/dagster/python_modules/dagster/dagster/_utils/__init__.py", line 220, in __hash__
    return hash(tuple(sorted(self.items())))
  File "/Users/ben/Documents/repos/dagster/python_modules/dagster/dagster/_utils/__init__.py", line 251, in __hash__
    return hash(tuple(self))
  File "/Users/ben/Documents/repos/dagster/python_modules/dagster/dagster/_utils/__init__.py", line 220, in __hash__
    return hash(tuple(sorted(self.items())))
  File "/Users/ben/Documents/repos/dagster/python_modules/dagster/dagster/_utils/__init__.py", line 220, in __hash__
    return hash(tuple(sorted(self.items())))
```

Reply:
As discussed offline, fixed by ensuring that the metadata classes contain `frozenlists` instead of normal lists.